handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -129,6 +131,14 @@ public SelectedListenerFailureBehavior selectedListenerFailureBehavior() {
         }
         this.mode = mode;
 
+        if (mode == SSL.SSL_MODE_SERVER) {
+            // To make it easier for users to replace JDK implemention with OpenSsl version we also use
+            // 'jdk.tls.rejectClientInitiatedRenegotiation' to allow disabling client initiated renegotiation.
+            // Java8+ uses this system property as well.
+            // See http://blog.ivanristic.com/2014/03/ssl-tls-improvements-in-java-8.html
+            rejectRemoteInitiatedRenegation =
```

Comment:
Should we just do this in a `static` block and save the result in a `DEFAULT_REJECT...` variable?